docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -0,0 +1,68 @@
+---
+---
+= Clustered Cache
+ifndef::sourcedir[]
+include::common.adoc[]
+endif::sourcedir[]
+
+ifdef::notBuildingForSite[]
+include::menu.adoc[]
+endif::notBuildingForSite[]
+
+== Introduction
+
+As with the standalone cache, in order to use the clustered EhCache you will need to configure Clustered cache manager and the cache.
```

Comment:
This sentence reads strange and I do not see the point of referring to the standalone cache.